master/getting-started/bare-metal/bare-metal.md

```diff
@@ -31,21 +32,16 @@ However, for host endpoints, Calico is more lenient; it only polices
 traffic to/from interfaces that it's been explicitly told about. Traffic
 to/from other interfaces is left alone.
 
-As of Calico v2.1.0, Calico applies host endpoint security policy both to traffic
-that is terminated locally, and to traffic that is forwarded between host
-endpoints.  Previously, policy was only applied to traffic that was terminated
-locally.  The change allows Calico to be used to secure a NAT gateway or router.
-Calico supports selector-based policy as normal when running on a gateway or router
-allowing for rich, dynamic security policy based on the labels attached to your
-workloads.
-
-> **Note**: If you have a host with workloads on it then traffic that is forwarded to
-> workloads bypasses the policy applied to host endpoints. If that weren't the
-> case, the host endpoint policy would need to be very broad to allow all
-> traffic destined for any possible workload.
+Host endpoint policy allows Calico to be used to secure a NAT gateway or router. Calico supports selector-based policy as normal when running on a gateway or router allowing for rich, dynamic security policy based on the labels attached to your
+workloads. Host endpoint policy applies to three types of traffic, traffic
+that is terminated locally, traffic that is forwarded between host endpoints, and traffic that is forwarded between host endpoint and workload endpoint on the same host. Users can specify if a given host endpoint policy applies to forwarded traffic or not by setting `applyOnForward` flag. 
+
```

Comment:
This is something I'm going to be discussing more in the upcoming style guide, but in general, I will be recommending to create more immediacy and specificity by using the imperative and the second person. So this second sentence would become, "Set the [`applyOnForward` flag](link) to specify if a given policy applies to forwarded traffic or not."

I am not sure just how to set this `applyOnForward` flag and a search in the documentation did not turn up anything. Hopefully, there is somewhere where this is discussed and you can link to it from here.

Reply:
OK I found it....it's in this PR! Great, so you can use a link like this then `/master/reference/calicoctl/resources/policy#spec`

Reply:
Good point.